src/components/Island.scss

```diff
@@ -6,4 +6,8 @@
   border-radius: var(--border-radius-m);
   padding: calc(var(--padding) * var(--space-factor));
   position: relative;
+
+  &.zen-mode {
+    box-shadow: none;
```

Comment:
Same as the other suggestion, let's animate this transition. It looks very sudden otherwise.